modules/backend/behaviors/RelationController.php

```diff
@@ -713,6 +720,26 @@ protected function makeViewWidget()
                     }
                 }
             }
+
+            /*
+             * Prepare the filter widget (optional)
+             */
+            if ($this->getConfig('view[filter]')) {
```

Comment:
Could you switch this to `if ($config->filter)` since that is actually the variable getting used later?

Reply:
Good catch. Yes, give me a minute and I'll update the PR.